examples/kitty/client/kitty.py

```diff
@@ -17,8 +17,13 @@
 token = 0
 reader_stream = None
 writer_stream = None
+card_ready = True
+# These are defaulted to true.
+# If you wish to disable, set in kitty.run()
+enable_i2c = True
+enable_nfs = True
 
-IP_ADDRESS = "172.16.0.2"
+IP_ADDRESS = "10.0.2.2"
```

Comment:
We should not hard-code the IP address nor the port number. The port number can have a default one but the IP address should be supplied when you run the script.